packages/typespec-vscode/test/extension/create-typespec.config.ts

```diff
@@ -0,0 +1,40 @@
+enum CreateProjectTriggerType {
+  Click = "RightClick",
+  Command = "CommandPalette",
+}
+
+type CreateConfigType = {
+  caseName: string;
+  triggerType: CreateProjectTriggerType;
+  templateName: string;
+  templateNameDescription: string;
+  isEmptyFolder: boolean;
+  expectedResults: string[];
+};
+
+const CreateTypespecProjectFolderPath = "../.vscode-test/CreateTypespecProject";
```

Comment:
this will fail if the `cwd` is different. And again please use the `temp` folder that every other package use for those kind of things(<pkgRoot>/temp>)

Maybe instead of `__dirname` all over the palce save `projectRoot` and `tempDir` variables pointing the the package root and the temp dir so you can reuse it

Reply:
Updated.